recipes/proxy-py/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "2.4.3" %}
+
+package:
+  name: proxy-py
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/p/proxy.py/proxy.py-{{ version }}.tar.gz
+  sha256: 6134e8f1282db1fd7fa1a4b7049e49307566851023b2ac312d9dd36e92f0c9b1
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  entry_points:
+    - proxy = proxy:entry_point
+  script_env:
+    - PYTHONIOENCODING=utf-8
+    - PYTHONLEGACYTEXTENCODING=utf-8
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - setuptools-scm >=6,!=7.0.0,!=7.0.1,!=7.0.2
+  run:
+    - python >=3.6
+
+test:
+  imports:
+    - proxy
+  commands:
+    - pip check
+    - proxy --help
+  requires:
+    - pip
+
+about:
+  home: https://github.com/abhinavsingh/proxy.py
+  summary: ⚡ Fast • 🪶 Lightweight • 0️⃣ Dependency • 🔌 Pluggable • 😈 TLS interception • 🔒 DNS-over-HTTPS • 🔥 Poor Mans VPN • ⏪ Reverse & ⏩ Forward • 👮🏿 Proxy Server framework • 🌐 Web Server framework • ➵ ➶ ➷ ➠ PubSub framework • 👷 Work acceptor & executor framework.
```

Comment:
Can we make this summary a bit shorter?

Reply:
down to a ascii string :stuck_out_tongue_closed_eyes:

Reply:
Nothing against the emoji use there but it was quite long and prone to break dashboards :-)